src/NServiceBus.Core/Notifications.cs

```diff
@@ -2,16 +2,10 @@
 
 namespace NServiceBus
 {
-    using Faults;
-
     /// <summary>
     /// Notifications.
     /// </summary>
     public class Notifications
```

Comment:
Don't we have to obsolete the whole thing now that the property is gone?

Reply:
There's definitely something off here because this class is now an empty stub. However, none of the public APIs around it has any kind of obsoletes, so it seems like something was overlooked when `ErrorNotifications` was being removed.